phpunit.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit
+    colors="true"
+    bootstrap="./vendor/autoload.php" type="coverage-text" target="tests/" charset="UTF-8">
+    
+    <testsuites>
+        <testsuite name="testing">
+      	    <directory>tests/</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
```

Comment:
Please have a look on this bundle for a filter example : https://github.com/Th3Mouk/YahooWeatherBundle/blob/master/phpunit.xml.dist#L21

Is this can be `<directory>./src</directory>` ?

IMO the previous code was good, just remove the useless exclusion rules.